integration-tests/environments/react-native/ios/RealmReactNativeTests.xcodeproj/project.pbxproj

```diff
@@ -270,8 +270,8 @@
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "set -e\nPLATFORM_NAME=ios\nWITH_ENVIRONMENT=\"../../../../node_modules/react-native/scripts/xcode/with-environment.sh\"\nREACT_NATIVE_XCODE=\"../../../../node_modules/react-native/scripts/react-native-xcode.sh\"\n\n/bin/sh -c \"$WITH_ENVIRONMENT $REACT_NATIVE_XCODE\"\n";
+			shellPath = "/bin/sh -l";
+			shellScript = "set -e\nPLATFORM_NAME=ios\nWITH_ENVIRONMENT=\"../../../../node_modules/react-native/scripts/xcode/with-environment.sh\"\nREACT_NATIVE_XCODE=\"../../../../node_modules/react-native/scripts/react-native-xcode.sh\"\nexport PROJECT_ROOT=\"$PROJECT_DIR/..\"\n/bin/sh -c \"$WITH_ENVIRONMENT $REACT_NATIVE_XCODE\"\n";
```

Comment:
Was this intentional? Just worried about the possibility of this being unintentionally wiped after a RN upgrade

Reply:
Yeah. I've contributed a patch upstream to change the default to be more friendly towards mono-repos: https://github.com/facebook/react-native/pull/35970

Reply:
But with [the changes I did to `.xcode.env`](https://github.com/realm/realm-js/pull/5330/files#diff-0369cbaa30691bc3629a31a58b70cf9b8a6a909cb2f486446315eb93d553b055R11-R12) I can actually revert the update of the shellPath.

Reply:
Cool!  Hopefully it gets approved :)